packages/experiments/src/components/Toggle/Toggle.styles.ts

```diff
@@ -58,16 +58,18 @@ export const ToggleStyles: IToggleComponent['styles'] = props => {
     checkedDisabled: {
       pillBackground: semanticColors.disabledBodyText,
       pillBorderColor: 'transparent',
-      pillJustifyContent: 'flex-end' as IToggleStyleVariablesTypes['pillJustifyContent'],
+      pillJustifyContent: 'flex-end',
 
       thumbBackground: semanticColors.disabledBackground,
 
       textColor: semanticColors.disabledText,
       textHighContrastColor: 'GrayText'
     },
 
-    styleVariables
-  });
+    ...styleVariables
```

Comment:
Did you confirm that this is correct? If so, can you add a TODO (or fix) Button too?

One thing we need to do is add a bunch of cases for Button and Toggle trying to style every portion of the component, but we can do that in another PR.

Reply:
I did not confirm this. Actually, having it as a separate PR might be a good thing, still need to make a couple of changes here before being able to merge, though.